### ISSUE TITLE
Demo example: user_ ID is empty

### ISSUE THREAD

**Issue by JellyZero** [matthewhartstonge/storage#65]

**Comment by matthewhartstonge** [comment]
@JellyZero got a file + line number as to where?

**Comment by JellyZero** [comment]
accessTokens ttable name data

**Comment by matthewhartstonge** [comment]
Having a look into this, from what I can tell the same issue would be present with the official example @ https://github.com/ory/fosite-example/blob/master/authorizationserver/oauth2.go#L81-L106

```go
// A session is passed from the `/auth` to the `/token` endpoint. You probably want to store data like: "Who made the request",
// "What organization does that person belong to" and so on.
// For our use case, the session will meet the requirements imposed by JWT access tokens, HMAC access tokens and OpenID Connect
// ID Tokens plus a custom field


// newSession is a helper function for creating a new session. This may look like a lot of code but since we are
// setting up multiple strategies it is a bit longer.
// Usually, you could do:
//
//  session = new(fosite.DefaultSession)
func newSession(user string) *openid.DefaultSession {
	return &openid.DefaultSession{
		Claims: &jwt.IDTokenClaims{
			Issuer:      "https://fosite.my-application.com",
			Subject:     user,
			Audience:    []string{"https://my-client.my-application.com"},
			ExpiresAt:   time.Now().Add(time.Hour * 6),
			IssuedAt:    time.Now(),
			RequestedAt: time.Now(),
			AuthTime:    time.Now(),
		},
		Headers: &jwt.Headers{
			Extra: make(map[string]interface{}),
		},
	}
}
```

The difference I've patched in via #67 sets the top level `DefaultSession` properties for subject and username, which then get persisted through into the other collections as expected for linking: https://github.com/matthewhartstonge/storage/pull/67/files#diff-c74895781e071378bb42d3292e00d6c8e2f89118298d33bc0bb17836d9575fcb
